act-rules-format.bs

```diff
@@ -315,31 +315,28 @@ The definition of a rule MUST always result in one of the following outcomes:
 
-The web is ever changing, and technologies are used in such diverse and creative ways that it is impossible to predict in advance, all the ways that accessibility issues can occur and all the ways they can be solved for. When writing ACT Rules, it is almost inevitable that exceptions will be overlooked during the design of a rule, or that new technologies will emerge that introduce new exceptions.
+This section is *non-normative*.
 
-This makes it important to be able to regularly test if the rule has the accuracy that is expected of it. This can be done by benchmark testing. In benchmark testing, the accuracy of a rule is measured by comparing its results to those obtained through accessibility expert testing.
+The web is ever changing, and technologies are used in such diverse and creative ways that it is impossible to predict in advance, all the ways that accessibility issues can occur and all the ways they can be solved for. When writing ACT Rules, it is almost inevitable that edge cases will be overlooked during the design of a rule, or that new technologies will emerge that introduce exceptions.
 
-The accuracy of a rule is the average between the false positives and false negatives, which are in turn calculated as follows:
+When a test result incorrectly indicates non-conformance to an accessibility requirement, this is known as a false positive. Opposite, when a rule incorrectly indicates conformance, this is a false negative. A percentage of false positives and false negatives can be calculated by comparing it to results from expert accessibility evaluations:
 
-- **False positives**: This is the percentage of [test targets](#output-test-target), that were failed by the rule, but were not failed by an accessibility expert.
+- **False positives**: This is the percentage of [test targets](#output-test-target), that were failed by the rule, but were conformanct to the [accessibility requirements](#structure-accessibility-requirements).
 
-- **False negatives**: This is the percentage of test targets, that were passed by the rule, but were failed by an accessibility expert.
+- **False negatives**: This is the percentage of test targets, that were passed by the rule, but were non-conformant to the [accessibility requirements](#structure-accessibility-requirements).
 
-There are several ways this can be done. For instance, accessibility test tools can implement a feature which lets users indicate that a result is in error, or pages that for which accessibility results are known, can be tested using ATT, and the results are compared. To compare results from ACT Rules to those of expert evaluations, [data aggregation](#output-aggregation) may be necessary.
+The possibility of false positives and false negatives with ACT rules makes it important to have a process in place for designing and maintaining accurate rules. An example of such a process is the [ACT Review Process](https://w3c.github.io/wcag-act-rules/review-process.html). 
```

Comment:
As per @nitedog's comments about adding "measured by the test cases provided", I think that we need to separately treat two different types of false positives/negatives here: 
* When an implementation has false positives/negatives when measuring against the test cases provided for an ACT rule
* When implementations of a rule live up to the test cases provided for a rule, but still has false positives/negatives compared to expert testing, this could indicate that the rule itself is inaccurate or that the test cases provided are not comprehensive - and THEN it is important with the process

To me it seems that the current section is a bit of a mix of the two, or at least leaves too much room to interpret it in both directions.

Reply:
In addition to this, shouldn't the phrase "designing and maintaining accurate rules" include the testing of the rules?  You have to test the implementation of the rules to find out their accuracy.